modules/search-service-api/src/main/java/org/opencastproject/search/api/SearchResult.java

```diff
@@ -252,7 +260,8 @@ public Map<String, Object> dehydrateEpisode() {
         DUBLINCORE, SearchResult.dehydrateDC(this.dublinCore),
         ORG, this.orgId,
         TYPE, this.type.name(),
-        MODIFIED_DATE, DateTimeFormatter.ISO_INSTANT.format(this.modified)));
+        MODIFIED_DATE, DateTimeFormatter.ISO_INSTANT.format(this.modified),
+        LIVE, this.live));
```

Comment:
So this puts the live flag into the index, but there's no explicit handling of the flag on the other (`rehydrate()`) end.  Even if it's not required (since the queries are looking at the index directly), I think we should reconstruct the live flag on the other end too.